content/javascript/concepts/dates/terms/setUTCDate/setUTCDate.md

```diff
@@ -0,0 +1,40 @@
+---
+Title: '.setUTCDate()'
+Description: 'Changes the day to a Date object in Coordinated Universal Time (UTC).'
+Subjects:
+  - 'Web Development'
+Tags:
+  - 'Date'
+  - 'Methods'
+  - 'Time'
+CatalogContent:
+  - 'paths/web-development'
+  - 'introduction-to-javascript'
+---
+
+The **`.setUTCDate()`** method changes the day to a Date object.
+
+## Syntax
+
+```pseudo
+myDate.setUTCDate(day)
+```
+
+The `.setUTCDate()` method receives a day argument representing a number between 1 to 31.
+
+## Example
+
+In this example, we create a Date object in a variable called `myDate`. We then use `.setUTCDate()` to modify the day to 10.
```

Comment:
```suggestion
In this example, a Date object is created in a variable called `myDate`. `.setUTCDate()` is then used to modify the day to 10.
```
try not to use pronouns

Reply:
In particular we avoid using first- or second-person pronouns in Docs.